moxin-frontend/src/app.rs

```diff
@@ -53,54 +57,68 @@ live_design! {
-                    padding: { top: 80 }
+                root = <View> {
+                    width: Fill,
+                    height: Fill,
+
+                    sidebar_menu = <View> {
+                        width: 100,
+                        flow: Down, spacing: 20.0,
+                        padding: { top: 80 }
 
-                    align: {x: 0.5, y: 0.0},
+                        align: {x: 0.5, y: 0.0},
 
-                    show_bg: true,
-                    draw_bg: {
-                        color: #EDEEF0,
-                    }
+                        show_bg: true,
+                        draw_bg: {
+                            color: #EDEEF0,
+                        }
 
-                    discover_tab = <SidebarMenuButton> {
-                        animator: {selected = {default: on}}
-                        label: "Discover",
-                        draw_icon: {
+                        tab1 = <SidebarMenuButton> {
+                            animator: {selected = {default: on}}
+                            label: "Discover",
+                            draw_icon: {
                             svg_file: (ICON_DISCOVER),
+                            }
                         }
-                    }
-                    chat_tab = <SidebarMenuButton> {
-                        label: "Chat",
-                        draw_icon: {
+                        tab2 = <SidebarMenuButton> {
+                            label: "Chat",
+                            draw_icon: {
                             svg_file: (ICON_CHAT),
+                            }
                         }
-                    }
-                    my_models_tab = <SidebarMenuButton> {
-                        label: "My Models",
-                        draw_icon: {
+                        tab3 = <SidebarMenuButton> {
+                            label: "My Models",
+                            draw_icon: {
                             svg_file: (ICON_MY_MODELS),
+                            }
                         }
                     }
-                }
 
-                application_pages = <View> {
-                    margin: 0.0,
-                    padding: 0.0,
+                    application_pages = <View> {
+                        margin: 0.0,
+                        padding: 0.0,
 
-                    flow: Overlay,
+                        flow: Overlay,
 
-                    width: Fill,
-                    height: Fill,
+                        width: Fill,
+                        height: Fill,
 
-                    discover_frame = <LandingScreen> {visible: true}
-                    chat_frame = <ChatScreen> {visible: false}
-                    my_models_frame = <MyModelsScreen> {visible: false}
+                        tab1_frame = <LandingScreen> {visible: true}
+                        tab2_frame = <ChatScreen> {visible: false}
+                        tab3_frame = <MyModelsView> {visible: false}
```

Comment:
Looks like we're reverting some renaming done recently by @joulei

Reply:
opps, i messed up the merge, fixing it.